src/client/phy-ue.cc

```diff
@@ -438,11 +435,25 @@ void PhyUe::Start() {
                 fft_dlpilot_counters_.CompleteSymbol(frame_id);
             if (pilot_fft_complete) {
               if (kPrintPhyStats) {
-                this->phy_stats_->PrintDlSnrStats(frame_id, ant_id);
+                this->phy_stats_->PrintDlSnrStats(frame_id);
               }
               if (kEnableCsvLog) {
```

Comment:
You removed all but 1 other kEnableCsvLog in phy ue.  Probably best to remove this one too.   You can just pass these into phy_stats and let it make the decision.